index.Rmd

```diff
@@ -24,26 +19,216 @@ options(width = 60)
+Der\*die P.I. kann bei einer Studie prinzipiell dieselben Funktionen ausführen wie ein Researcher. Der Hauptunterschied zwischen einem P.I.- und einem Researcher-Konto besteht darin, dass Researcher die Möglichkeit haben, Routine-E-Mails vom System über ihre Studie zu erhalten. Dies sind z. B. Benachrichtigungen, wenn sich Versuchspersonen für die Studie anmelden oder abmelden. Der\*die P.I. hat keine Möglichkeit, solche E-Mails zu erhalten. Somit sollten Personen, die für die Datenerhebung (insbesondere im Labor) zuständig sind, einen Researcher-Account erhalten, während Projektleiter\*innen  einen P.I. Account erhalten sollten. Für jede Studie muss mindestens ein Researcher und ein P.I. ausgewählt werden. 
+
+
+## Kann ich die Website auch auf Deutsch sehen?
+
+Nein, für Forscher\*innen ist die Website leider nur auf Englisch verfügbar. Die Website wird den Versuchspersonen aber, sofern nicht anders ausgewählt, auf Deutsch angezeigt.
+
+
+# Studien einstellen und verwalten
+
+## Studien einstellen
+
+Sona Systems unterscheidet fünf Arten von Studien, wobei jede Art entweder mit VP-Stunden („Credit“) oder mit einer Bezahlung („Paid“; auch Gutscheine o.Ä. möglich) vergütet werden kann. Studien, in denen Teilnehmer\*innen zwischen VP-Stunden und Bezahlung wählen können, werden als „Credit“-Studie angelegt. Die alternative Aufwandsentschädigung wird dann in der Beschreibung der Studie erwähnt. 
+
+-	**Standard Study:** normale Laborstudie (nicht online) 
+-	**Multi-Part Standard Study:** normale Laborstudie (nicht online), die aus mindestens zwei Teilen besteht 
+-	**Online External Study:** Online-Studie, die auf einer anderen Website durchgeführt wird als Sona (z. B. SoSci Survey, Lab.js usw.). Dies ist bei den meisten Online-Studien der Fall. 
+-	**Multi-Part Online External Study:** Online-Studie, die auf einer anderen Website durchgeführt wird als Sona und mehrere Messzeitpunkte hat
+-	**Online Internal Survey Study:** Online-Befragung innerhalb von Sona 
+
+<br>
+Um Ihre Studie auf Sona einzustellen, gehen Sie folgendermaßen vor.
+
+1.	Loggen Sie sich mit Ihrem Forschenden-Account auf https://uni-mainz-jgu.sona-systems.com ein.
+2.	Wählen Sie entweder `Add New Study` im Menüstreifen oder `+ Create a new Study`.
+
+```{r echo = FALSE, out.width='92%', fig.align='center'}
+knitr::include_graphics('images/Folie10.png')
+```
+
+3.	Wählen Sie die passende Art von Studie aus und klicken sie auf `Continue`.
+4.	Nun geben Sie alle Informationen zu Ihrer Studie ein. Die genauen Erklärungen für die Eingabefelder sind in den folgenden Screenshots aufgeführt. 
+5.	Speichern Sie Ihre Eingabe, indem Sie auf den grünen Knopf `Add this Study` klicken. Die Studie wird den Versuchspersonen nur angezeigt, wenn sie sowohl „aktiv“ als auch „approved“ ist. Sie können die Studie selbst auf „aktiv“ setzen, aber nur die Administration kann Ihre Studie genehmigen („approve“). 
+
+
+```{r echo = FALSE, out.width='92%', fig.align='center'}
```

Comment:
Let's see how these images render, we might have to come up with a better way to "chain" images after each other. But it's good for now :)